dsgrid/registry/dimension_mapping_registry_manager.py

```diff
@@ -546,3 +546,18 @@ def show(
         if return_table:
             return table
         display_table(table)
+
+    def try_get_mapping(self, from_dimension, from_version, to_dimension, to_version):
+        """Return the mapping config and version for from and to dimension, if one exists."""
+        from_id = from_dimension.model.dimension_id
```

Comment:
I think from_dimension and to_dimension here are configs? Shouldn't they already contain version info?

Reply:
Dimension configs and models do not contain versions.